wow_library/include/Attributes.hpp

```diff
@@ -17,7 +17,7 @@ struct Special_stats
                   double haste = 0, double damage_mod_physical = 0, double stat_multiplier = 0,
                   double bonus_damage = 0, double crit_multiplier = 0, double spell_crit = 0,
                   double damage_mod_spell = 0, double expertise = 0, double sword_expertise = 0, double mace_expertise = 0, double axe_expertise = 0,
-                  int gear_armor_pen = 0, double ap_multiplier = 0, double attack_speed = 0)
+                  int gear_armor_pen = 0, double ap_multiplier = 0, double attack_speed = 0, double str_multiplier = 0)
```

Comment:
arguably, stat_multiplier should become a mere "agi_multiplier" now - just adding a multiplier like this is less infilatrative, and therefore better, though